ui/app/manager/src/pages/page-account.ts

```diff
@@ -348,52 +353,58 @@ export class PageAccount extends Page<AppStateKeyed>  {
         `;
     }
 
-    /**
-     * HTML callback function when any of the password input fields change.
-     * Checks whether {@link _passwordElem} and {@link _repeatPasswordElem} are the same,
-     * and updates the {@link user} object with the new password.
-     */
-    protected _onPasswordChanged(user: UserModel): boolean {
-        const password = this._passwordElem?.nativeValue;
-        const repeatPassword = this._repeatPasswordElem?.nativeValue;
-        if(password && repeatPassword) {
-
-            if(password !== repeatPassword) {
-                const error = i18next.t("passwordMismatch");
-                this._repeatPasswordElem?.setCustomValidity(error);
-                user.password = "";
-
-            } else {
-                this._repeatPasswordElem?.setCustomValidity(undefined);
-                user.password = password;
-                return true;
-            }
-        } else {
-            console.warn("Could not update password; some fields are empty;", password, repeatPassword);
-        }
-        return false;
-    }
-
     /**
      * HTML callback function for when the 'Save' button is clicked.
      * Checks if the input fields are valid, and calls the {@link _updateUser} function.
      */
     protected _onSaveBtnClick(_e: OrInputChangedEvent) {
-        if(this._user && !this._isInvalid()) {
+        if (this._user && !this._isInvalid()) {
             this._updateUser(this._user);
         } else {
             console.warn("The fields are invalid!");
             showSnackbar(undefined, "saveUserFailed");
         }
     }
 
+    /**
+     * HTML callback function for when the 'Reset Password' button is clicked.
+     */
+    protected _onResetPasswordBtnClick(_e: OrInputChangedEvent) {
+        if (this._user && !this._isInvalid()) {
+            this._requestPasswordReset()
```

Comment:
Calling this function needs an exception handler. (as the function itself doesn't handle errors)